nautobot_floor_plan/models.py

```diff
@@ -79,6 +81,37 @@ def get_svg(self, *, user, base_url):
         """Get SVG representation of this FloorPlan."""
         return FloorPlanSVG(floor_plan=self, user=user, base_url=base_url).render()
 
+    def save(self, *args, **kwargs):
+        """Override save in order to update any existing tiles."""
+        if not self.created:
+            super().save(*args, **kwargs)
+            return
+        # Get origin_seed pre/post values
+        initial_instance = self.__class__.objects.get(pk=self.pk)
+        x_initial = initial_instance.x_origin_seed
+        y_initial = initial_instance.y_origin_seed
+        x_updated = self.x_origin_seed
+        y_updated = self.y_origin_seed
+
+        super().save(**kwargs)
+
+        if x_initial != x_updated or y_initial != y_updated:
+            self.update_tile_origins(x_initial, x_updated, y_initial, y_updated)
```

Comment:
Perhaps?

```suggestion
        if self.present_in_database:
            # Get origin_seed pre/post values
            initial_instance = self.__class__.objects.get(pk=self.pk)
            x_initial = initial_instance.x_origin_seed
            y_initial = initial_instance.y_origin_seed
            changed = (x_initial != self.x_origin_seed or y_initial != self.y_origin_seed)
        else:
            changed = False

        super().save(*args, **kwargs)

        if changed:
            self.update_tile_origins(x_initial, self.x_origin_seed, y_initial, self.y_origin_seed)
```

Reply:
looks good! sorry for not seeing present_in_database before, found it and reviewing more in the best practice documentation now :)